Improve copywriting (#279)
openfisca/openfisca-doc: source/index.md

# <i class="fas fa-home"></i> Before you start

## What is OpenFisca

[OpenFisca](https://openfisca.org) is an open-source engine to write rules as code.

Describe your tax and benefit system, provide a situation as input (i.e income), ask for a calculation as output (i.e. income tax), and get your results!

## Who uses OpenFisca

> * **Economists and lawmakers**:

Reviewer: If we want to add a newline, then let's use a subtitle markup 🙂 
```suggestion
### Economists and lawmakers

```

Author: Oh Okay. will do 👍

Economists and lawmakers make use of OpenFisca to calculate the effects of policies. By combining them with survey data to simulate the impact of reform on a given government’s budget and a population’s standard of living.

* **Developers and companies**:
Developers and companies use Openfisca to effortlessly create web applications based on simulation results with the help of the OpenFisca [web API](openfisca-web-api/index.md). You can build a great significant variety of services by coding formulas, hosting your instances and building your extensions.

* **Public administrations**:
Stop building your micro-simulation software and tax & benefit calculators from scratch. Join OpenFisca, contribute to OpenFisca, collaborate with other administrations and reduce costs paid by the taxpayers.

> ## Ways to use OpenFisca

Reviewer: The three listed points are sequential and not alternatives 🙂 

```suggestion
## Way to use OpenFisca
```

Author: okay Got it

* [Build a new tax and benefit system](coding-the-legislation/bootstrapping_a_new_country_package.md) if it doesn’t exist already.
* [Contribute](contribute/index.md) to an existing system by adding or improving elements of the legislation.

> With many helpful OpenFisca tools, you can turn legal code into OpenFisca code which is a subset of the Python programming language:

Reviewer: Here, the suggested change does not work: the intention is really to say that OpenFisca is “a subset of the Python programming language with many helpful tools”. It is not about “turning legal code into OpenFisca code with many helpful tools”. Do you have another suggestion to make that sentence clearer? 🙂

Author: Then it would be more approriate to add the line " OpenFisca is a subset of the Python programming language with many helpful tools” at the beginning of the doc.
And replace the above  sentence with maybe: " Next, you will turn the legal code to OpenFisca code by following the below steps" or something along that logic?

Reviewer: I see your point, thanks for this suggestion!

I don't think we can say that OpenFisca _is_ a subset of Python. The OpenFisca [_DSL_](https://en.wikipedia.org/wiki/Domain-specific_language) is “a subset of the Python programming language with many helpful tools”, but OpenFisca is more than its DSL.

This is really not clear indeed 😄 Thank you so much for identifying this rough edge! What would you think of the following:

```suggestion
Then, you will make legal code executable by writing it in the OpenFisca [DSL](https://en.wikipedia.org/wiki/Domain-specific_language), which is a subset of the Python programming language with dedicated functions and tools specifically targeted at modelling rules:
```

Are you trying to assess the impact of a new housing tax on behalf of the OECD? Find your government's open survey data and use it with the OpenFisca to simulate the effect of that tax reform on the poorest 20% of a country.

### 3 - Run Simulations

With OpenFisca there are two ways to calculate the effect of the rules modelled in a country package on a given input data:

*With the results of your simulation, efficiently build a web application. Using the [web API](openfisca-web-api/index.md).
**Using the  [Python API](openfisca-python-api/index.md) data scientist can utilize a large dataset to perform a survey or apply changes to your system dynamically.

The output of this simulation will be either in Python objects or JSON. To represent these results in a graphical or visualized format you can make use of python libraries for ex:([plot.ly](https://plot.ly) for instance will get you those nice charts you’ve seen elsewhere).

Please make sure you read our [license](license.md) before publishing results based on OpenFisca.

## Things OpenFisca won’t do for you

> * **Behaviour-based analysis.**

Author: Done 👍

* **Human decisions.**
This might seem obvious, but it is worth reiterating: any process that includes human judgement can not be completely automated. Human decisions can be modelled as input variables, but OpenFisca does not aim at doing machine learning on past decisions for probabilistic results, for example.
* **Zero effort modelling.**
While our community is strong and you can often benefit from its shared efforts, OpenFisca is contributive: if the legislation you need is not described yet, you’re the best person to add it (take a look at our [contribution guidelines](contribute/index.md) first).

* **Magical legislation parsing.**
OpenFisca is a framework for humans to collaborate on translating rules into code. It does not ingest lawyer speak(abstruse jargon of lawyers) and automatically produces developer speak(terminologies used by developers). Regulations are ambiguous and need human interpretation before being codified.

* **Textual comparisons.**
OpenFisca shines in dealing with numbers: enumerated values are a later addition. Hence support for serving and testing with _string values_ is not built-in.

> ## Feeling lost 😟?

Reviewer: Lacking clear guidelines for using emojis, let's avoid having a lonely one 😉 
```suggestion
## Feeling lost?
```

Author: 😅 okay. Noted.
